tests/specializations/mappings/test_mappings.py

```diff
@@ -13,3 +13,43 @@
 # along with Blue Brain Nexus Forge. If not, see <https://choosealicense.com/licenses/lgpl-3.0/>.
 
 # Placeholder for the generic parameterizable test suite for mappings.
+
+
+import pytest
+from contextlib import nullcontext as does_not_raise
+
+from kgforge.core.archetypes.mapping import MappingType
+from kgforge.specializations.mappings import DictionaryMapping
+from utils import full_path_relative_to_root
+
+mapping_url_valid = "https://raw.githubusercontent.com/BlueBrain/nexus-forge/master/examples" \
+                    "/configurations/nexus-store/file-to-resource-mapping.hjson"
+
+mapping_path_valid = full_path_relative_to_root(
+    "tests/data/nexus-store/file-to-resource-mapping.hjson"
+)
+
+
+@pytest.mark.parametrize(
+    "source, exception",
+    [
+        (mapping_path_valid, does_not_raise()),
+        (mapping_url_valid, does_not_raise()),
+    ],
+)
+def test_mapping_load_no_mapping_type(source, exception):
+    with exception:
+        mapping = DictionaryMapping.load(source)
+
+
+@pytest.mark.parametrize(
+    "source, mapping_type, exception",
+    [
+        (mapping_path_valid, MappingType.FILE, does_not_raise()),
+        (mapping_url_valid, MappingType.URL, does_not_raise()),
+        ("i", MappingType.URL, pytest.raises(Exception)),
```

Comment:
you can add more of the exception cases, to complete the testing (file path being passed as a url, text being fiven as file, url as text, etc.)

Reply:
I hadn't yet because there was no usage of these, but sure

Reply:
I've added some test cases